### PR TITLE
[OAP-1653][OAP-Cache]Keep consistency on 'enabled' of OapConf configu…

### PR DESCRIPTION
**Pull request by HongW2019**
…rations and be compatible with 'enable'

## What changes were proposed in this pull request?

Fix : #1653 

Refine OapConf and achieve compatibilty with the old configs if users ignore the new configs

1. Add   .executor to to notify users corresponding size setting is for per executor
2. Modify .fiberCache to .cache
3. Change .enable to .enabled

OLD | NEW
-- | --
  | Add   .executor
spark.sql.oap.fiberCache.offheap.memory.size | spark.executor.sql.oap.cache.offheap.memory.size
spark.sql.oap.fiberCache.persistent.memory.initial.size | spark.executor.sql.oap.cache.persistent.memory.initial.size
spark.sql.oap.fiberCache.persistent.memory.reserved.size | spark.executor.sql.oap.cache.persistent.memory.reserved.size
spark.sql.oap.cache.guardian.memory.size | spark.executor.sql.oap.cache.guardian.memory.size
spark.sql.oap.cache.external.client.pool.size | spark.executor.sql.oap.cache.external.client.pool.size
  | Modify .fibercache to .cache
spark.sql.oap.fiberCache.use.offheap.ratio | spark.sql.oap.offheap.cache.ratio
spark.sql.oap.dataCache.use.fiberCache.ratio | spark.sql.oap.cache.dataCache.ratio
spark.sql.oap.fiberCache.table.list.enable | spark.sql.oap.cache.table.list.enabled
spark.sql.oap.fiberCache.table.list | spark.sql.oap.cache.table.list
spark.sql.oap.fiberCache.memory.manager | spark.sql.oap.cache.memory.manager
spark.sql.oap.fiberCache.persistent.memory.config.file | spark.sql.oap.cache.persistent.memory.config.file
spark.sql.oap.fiberCache.size | ----no longer used and   deleted----
spark.sql.oap.fiberCache.stats | ----no longer used and   deleted----
  | Change .enable to .enabled (48/50 of Spark SQLConf use .enabled)
spark.sql.oap.orc.enable | spark.sql.oap.orc.enabled
spark.sql.oap.parquet.enable | spark.sql.oap.parquet.enabled
spark.sql.oap.index.data.cache.separation.enable | spark.sql.oap.index.data.cache.separation.enabled
spark.sql.oap.fiberCache.table.list.enable | spark.sql.oap.cache.table.list.enabled
spark.sql.oap.parquet.data.cache.enable | spark.sql.oap.parquet.data.cache.enabled
spark.sql.oap.orc.data.cache.enable | spark.sql.oap.orc.data.cache.enabled
spark.sql.oap.orc.binary.cache.enable | spark.sql.oap.orc.binary.cache.enabled
spark.sql.oap.parquet.index.enable | spark.sql.oap.parquet.index.enabled
spark.sql.oap.cache.external.read.only.enable | spark.sql.oap.cache.external.read.only.enabled
spark.sql.oap.index.statistic.externalsorter.enable | spark.sql.oap.index.statistic.externalsorter.enabled
spark.sql.oap.data.fiber.cache.compress.enable | spark.sql.oap.data.fiber.cache.compress.enabled
spark.sql.oap.memkind.conservative.enable | spark.sql.oap.memkind.conservative.enabled

### CODE REVIEW

**Comment by zhixingheyi-tian** [comment]
@yao531441 

Would you please have a review?

**Review by yao531441 — COMMENTED** [review]
LGTM！

File: oap-cache/oap/src/main/scala/org/apache/spark/sql/execution/datasources/HadoopFsRelationOptimizer.scala [hunk 1]
```diff
@@ -59,7 +59,8 @@ object HadoopFsRelationOptimizer extends Logging {
       // 2. canUseIndex: OAP_PARQUET_ENABLED is true and hasAvailableIndex.
       // Other scenarios still use ParquetFileFormat.
       case _: ParquetFileFormat
-        if relation.sparkSession.conf.get(OapConf.OAP_PARQUET_ENABLED) =>
+        if relation.sparkSession.conf.get(OapConf.OAP_PARQUET_ENABLED) &&
+          relation.sparkSession.conf.get(OapConf.OAP_PARQUET_ENABLE) =>
```
Comment (zhixingheyi-tian) [inline oap-cache/oap/src/main/scala/org/apache/spark/sql/execution/datasources/HadoopFsRelationOptimizer.scala:63]: Fix compatibility reading

Comment (HongW2019) [reply oap-cache/oap/src/main/scala/org/apache/spark/sql/execution/datasources/HadoopFsRelationOptimizer.scala:63]: hava fixed in  #1868

**Pull request merged**